### DISCUSSION TITLE
Enabling/disabling scrobbling per source

### DISCUSSION THREAD

**Discussion by sheriferson** [rekkyrosso/ampcast#3] [Ideas]
Hi, first of all, I came via a post you made on Reddit, been trying ampast with my Plex library, and it's a lovely app. Thanks for building it.

What do you think about enabling/disabling scrobbling per service?
My situation (I imagine is common to a few) is I have a Plex music library with Tidal integration. If I am using ampcast on Desktop, I would use Plexamp when away from my computer, on a mobile device, etc. I currently have Plex scrobble everything to Last.fm, and if I want to use ampast to play music from both Plex and Tidal, the scrobbling situation is a bit messy.

Option 1 is Plex scrobbles and ampcast doesn't, so Tidal music played with ampcast won't get scrobbled. Option 2 is I let ampast scrobble in which case I might (probably?) get duplicate scrobbles when listening to music from Plex library (both Plex and ampcast scrobbling). Option 3 I turn off Plex scrobbling, in which case none of the music I listen to anywhere but ampcast gets scrobbled. None of these options is great.

Ideally I can set ampcast to scrobble only Tidal music, and that solves everything. Curious what you think.

**Comment by sheriferson** [comment]
Wow, I should've maybe turned on Last.fm integration in ampcast first to see that this is already possible. Mea culpa.

**Reply by rekkyrosso** [reply]
No worries mate.

I use my app a lot and I had the same problem you reported so I fixed it ages ago.

**Reply by rekkyrosso** [reply]
Ampcast also checks with lastfm before scrobbling to make sure it's not a duplicate.
